aspnet/fundamentals/logging.rst

```diff
@@ -73,23 +96,37 @@ Exception error
 
 .. note:: Some loggers, such as the built-in ``ConsoleLogger`` used in this article, will ignore the ``eventId`` parameter. If you need to display it, you can include it in the message string. This is done in the following sample so you can easily see the eventId associated with each message, but in practice you would not typically include it in the log message.
```

Comment:
Are we intending the docs samples not to show any output as a result of working with event ids, then? I was thinking the reader would get more out of seeing them in action, but we're not showing any logging frameworks here that have real support for them.